src/WebApi/appsettings.json

```diff
@@ -5,8 +5,12 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  // NOTE: If any of your connection strings real passwords, you should delete them and put them in the user secrets file instead.
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=CleanArchitecture;Trusted_Connection=True;MultipleActiveResultSets=true;TrustServerCertificate=True"
+    // Local DB
+    //"DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=CleanArchitecture;Trusted_Connection=True;MultipleActiveResultSets=true;TrustServerCertificate=True"
+    // Docker DB
+    "DefaultConnection": "Server=localhost,1433;Initial Catalog=CleanArchitecture;Persist Security Info=False;User ID=sa;Password=yourStrong(!)Password;MultipleActiveResultSets=True;TrustServerCertificate=True;Connection Timeout=30;"
```

Comment:
How about just leave this blank, and put this in the Development.json?

The idea is that local development (might) work, but production will never work without the proper config.

Reply:
Good idea.

Reply:
Done